viite-backend/viite-main/src/main/scala/fi/liikennevirasto/viite/dao/RoadNetworkDAO.scala

```diff
@@ -420,83 +420,103 @@ class RoadNetworkDAO extends BaseDAO {
     Q.queryNA[MissingRoadwayPoint](query).iterator.toSeq
   }
 
+  private val commonFromClause =
```

Comment:
Joku brief olis selkeyttävä, esim.
"Gets valid CPs joined with related RPs, and valid roadways"